src/pages/Event/Event2023FallMissions.tsx

```diff
@@ -65,8 +120,9 @@ const Event2023FallMissions = () => {
       />
       <div css={{ height: "30px" }} />
       <CreditAmountStatusContainer />
-      <MissionContainer />
-      <MissionContainer />
+      {quests?.map((e, i) => (
+        <MissionContainer key={e.id} quest={e} />
+      ))}
```

Comment:
함수 내에서 `i`가 사용되지 않는 것 같습니다 !

```suggestion
      {quests?.map((quest) => (
        <MissionContainer key={quest.id} quest={quest} />
      ))}
```